docs/concepts/models.md

```diff
@@ -442,15 +414,19 @@ except ValidationError as e:
     """
 ```
 
-## Helper functions
+## Validating data
 
-*Pydantic* provides three `classmethod` helper functions on models for parsing data:
+!!! note
```

Comment:
Note moved from below. Nothing changed.

Reply:
Hmm I'm tempted to leave this down below, as it might lead folks to skip past a simple explanation of `model_validate_json`... up to you though, I don't have a strong opinion

Reply:
Ah yes did not pay attention to the fact this note was related to JSON. I initially moved it because it was next to two other notes, making the section a bit overloaded. I'll move it right after the `model_validate_json`